.travis.yml

```diff
@@ -9,6 +9,9 @@ before_install:
   - wget 'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-linux-x86_64.tar.bz2'
   - tar xf phantomjs-1.9.7-linux-x86_64.tar.bz2
   - export PATH=$PATH:phantomjs-1.9.7-linux-x86_64/bin/
+  - psql -c 'CREATE DATABASE "accounts-testing"' -U postgres
```

Comment:
I'm using the default test database name here instead of `travis_ci_test` like [tutor-server](https://github.com/openstax/tutor-server/blob/master/.travis.yml#L30).  Not completely sure why it was set to `travis_ci_test` in tutor-server.